simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountListRegistryFragment.kt

```diff
@@ -198,12 +196,14 @@ class AccountListRegistryFragment : Fragment(R.layout.account_list_registry) {
           }
           newText.equals("NYPL", ignoreCase = true) -> {
             this@AccountListRegistryFragment.accountListAdapter.filterList { account ->
-              account.title.contains("New York Public Library", ignoreCase = true)
+              account != null &&
+                account.title.contains("New York Public Library", ignoreCase = true)
```

Comment:
Ha, I forgot this was in here. It's fine to keep it, but for Palace, it will be more of an Easter egg than anything actually useful.